processing/src/main/java/org/apache/druid/query/filter/InDimFilter.java

```diff
@@ -526,6 +526,7 @@ public int nextInt()
     };
   }
 
+  @SuppressWarnings("ReturnValueIgnored")
```

Comment:
Will this suppression work if it's moved down to the single line `values.contains(null)`?

If not, consider adding a comment here about what line needs the ReturnValueIgnored suppression. It might not be obvious to people.

Reply:
the annotation can only be put on a method or an assignment